compiler/rustc_ast_pretty/src/pprust/state/item.rs

```diff
@@ -742,13 +744,16 @@ impl<'a> State<'a> {
                     self.print_lifetime_bounds(bounds);
                 }
             }
-            ast::WherePredicate::EqPredicate(ast::WhereEqPredicate { lhs_ty, rhs_ty, .. }) => {
+            ast::WherePredicateKind::EqPredicate(ast::WhereEqPredicate {
+                lhs_ty, rhs_ty, ..
+            }) => {
                 self.print_type(lhs_ty);
                 self.space();
                 self.word_space("=");
                 self.print_type(rhs_ty);
             }
         }
+        self.ann.post(self, AnnNode::SubItem(id));
```

Comment:
Any specific reason for adding this?

Reply:
Oh, it's a rebase error too

Reply:
I think I did this because I found the other places did this to the `NodeId` (`print_foreign_item` and `print_assoc_item`) so I imitated them.

It seems not the case because `WherePredicate` is not a sub-item.